roles/hosted_engine_setup/tasks/initial_clean.yml

```diff
@@ -56,12 +60,22 @@
         src: /usr/share/abrt/conf.d/plugins/vmcore.conf,
         dest: /etc/abrt/plugins/vmcore.conf
       }
-    when: abrt_exists.stat.exists
+    when: abrt_vdsm_config.stat.exists and abrt_installed_config.stat.exists
   - name: Restart abrtd service
     service:
       name: abrtd
       state: restarted
-    when: abrt_exists.stat.exists
+    when: abrt_vdsm_config.stat.exists and abrt_installed_config.stat.exists
+  - name: Remove vdsm's abrt config files
+    file:
+      state: absent
+      path: "{{ item }}"
+    with_items:
+      - /etc/abrt/abrt-action-save-package-data.conf
+      - /etc/abrt/abrt.conf
+      - /etc/abrt/plugins/CCpp.conf
+      - /etc/abrt/plugins/vmcore.conf
+    when: abrt_vdsm_config.stat.exists and not abrt_installed_config.stat.exists
```

Comment:
Wouldn't it be better to encapsulate `Restore initial abrt config files`, `Restart abrtd service`  and `Remove vdsm's abrt config files` into a `block` and use the condition for the whole block?

Reply:
`Remove vdsm's...` uses a different condition - notice the `not`, so the block would include only the first two. Do you think it's still worth it?

Reply:
Ahh sorry, missed that. So the I'd say stay as is, just please reformat the conditions